src/basic/cgroup-util.c

```diff
@@ -2401,7 +2401,7 @@ bool cg_is_unified_systemd_controller_wanted(void) {
         /* If the unified hierarchy is requested in full, no need to
          * bother with this. */
         if (cg_is_unified_wanted())
-                return 0;
+                return true;
```

Comment:
Ah, confusion on my side. Sure, if we have a fully unified system, the systemd controller needs no special treatment. Will turn it into `false` though.